packages/angularjs-extras/src/controllers/panel.js

```diff
@@ -34,8 +34,8 @@
 
       // Digest Model Bindings
       $scope.model = null
-      if ($element.attr('ng-model')) {
-        $scope.$parent.$watch($element.attr('ng-model'), function (model) {
+      if ($element.attr('data-ng-model')) {
```

Comment:
This should check for both, as technically both are valid possibilities.

Reply:
See other comments about this.